src/filters.js

```diff
@@ -30,8 +35,44 @@ const makeFilter = (
     values: Array.isArray(values) ? values : [values],
   });
 
-export default (params: any): FilterCreators => Object.keys(FILTERS).reduce(
-  (filters: FilterCreators, filter: string): FilterCreators =>
-    ({ ...filters, [filter]: makeFilter(FILTERS[filter], params) }),
-  {},
-);
+export const createFilter = (params: any): FilterCreators =>
+  Object.keys(FILTERS).reduce(
+    (filters: FilterCreators, filter: string): FilterCreators => ({
+      ...filters,
+      [filter]: makeFilter(FILTERS[filter], params),
+    }),
+    {},
+  );
+
+// todo make unit tests
+export const doesSatisfyToQueryFilters = (
+  item: Object,
+  queryFilters: Array<QueryFilter>,
+): boolean =>
+  queryFilters
+    .map((queryFilter: QueryFilter): boolean => {
```

Comment:
Shouldn't this be `some` instead of `map`?

Also, I'm not a huge fan of the name.  I don't understand when you would use this.